hubble-relay/cmd/serve/serve.go

```diff
@@ -141,6 +176,11 @@ func New(vp *viper.Viper) *cobra.Command {
 		false,
 		"Disable TLS for the server and allow clients to connect over plaintext.",
 	)
+	flags.Bool(
+		keyTLSServerMTLSEnforced,
+		false,
+		"Enforces clients to use mTLS. This option requires providing CA files in tls-hubble-server-ca-files",
+	)
```

Comment:
Is this flag needed? In other words, is there a use case where we want mTLS enabled but not required?

Reply:
I removed this flag, as after introducing the ``tls-relay-client-ca-files`` flag it is no longer needed. Originally this flag was for keeping backwards compatibility when relay was exposed over tls but indecure connections were still allowed. Right now it can be done by not providing  ``tls-relay-client-ca-files``